modules/monitoring-silencing-alerts.adoc

```diff
@@ -27,36 +27,64 @@ To silence a specific alert:
 
 * In the *Administrator* perspective:
 
-. Navigate to the *Observe* -> *Alerting* -> *Alerts* page of the {product-title} web console.
+. Go to *Observe* -> *Alerting* -> *Alerts* in the {product-title} web console.
 
-. For the alert that you want to silence, select the {kebab} in the right-hand column and select *Silence Alert*. The *Silence Alert* form will appear with a pre-populated specification for the chosen alert.
+. For the alert that you want to silence, click {kebab} and select *Silence Alert* to open the *Silence Alert* page with a default configuration for the chosen alert.
 
-. Optional: Modify the silence.
+. Optional: Change the default configuration details for the silence.
++
+[NOTE]
+====
+You must add a comment before saving a silence.
+====
 
-. You must add a comment before creating the silence.
-
-. To create the silence, select *Silence*.
+. To save the silence, click *Silence*.
 
 * In the *Developer* perspective:
 
-. Navigate to the *Observe* -> *<project_name>* -> *Alerts* page in the {product-title} web console.
+. Go to *Observe* -> *<project_name>* -> *Alerts* in the {product-title} web console.
+
+. If necessary, expand the details for the alert by selecting *>* next to the alert name.
+
+. Click the alert message in the expanded view to open the *Alert Details* page for the alert.
+
+. Click *Silence Alert* to open the *Silence Alert* page with a default configuration for the alert.
+
+. Optional: Change the default configuration details for the silence.
++
+[NOTE]
+====
+You must add a comment before saving a silence.
+====
+
+. To save the silence, click *Silence*.
 
-. Expand the details for an alert by selecting *>* to the left of the alert name. Select the name of the alert in the expanded view to open the *Alert Details* page for the alert.
+To silence a set of alerts by creating a silence configuration in the *Administrator* perspective:
 
-. Select *Silence Alert*. The *Silence Alert* form will appear with a prepopulated specification for the chosen alert.
+. Go to *Observe* -> *Alerting* -> *Silences* in the {product-title} web console.
 
-. Optional: Modify the silence.
+. Click *Create Silence*.
 
-. You must add a comment before creating the silence.
+. On the *Create Silence* page, set the schedule, duration, and label details for an alert.
++
+[NOTE]
+====
+You must add a comment before saving a silence.
+====
 
-. To create the silence, select *Silence*.
+. To create silences for alerts that match the labels that you entered, click *Silence*.
 
-To silence a set of alerts by creating an alert specification in the *Administrator* perspective:
+To silence a set of alerts by creating a silence configuration in the *Developer* perspective:
```

Comment:
see from https://github.com/openshift/openshift-docs/pull/63944/files#r1312440037, change to below format is fine
```
   In the Developer perspective:

```

Reply:
Fixed